src/index.js

```diff
@@ -98,6 +98,20 @@ const exported = {
         config.ACCESS_TOKEN = token;
     },
 
+    /**
+     * Sets a custom load tile function for url that starts with custom://
+     *
+     * @var {Function} loadFn - a function that receives
+     * (requestParameters: RequestParameters, callback: ResponseCallback<any>) and calls the callback with the tile buffer
+     * or with a Error instance
+     * @example
+     * mapboxgl.loadTilesFunction = (requestParameters, callback) => callback(null, tileBuffer, null, null);
+     */
+
+    set loadTilesFunction(loadFn: Function) {
+        config.LOAD_TILES_FUNCTION = loadFn;
```

Comment:
Another issue in CircleCI is that this property is not defined in the Config type in config.js.

Reply:
I followed the `ACCESS_TOKEN` pattern, I don't mind changing this, let me know what you think I should use...